l10n/templates/strings.properties

```diff
@@ -81,5 +88,10 @@ edit_topsites_showmore_button=Show more
 edit_topsites_showless_button=Show less
 edit_topsites_done_button=Done
 edit_topsites_pin_button=Pin this site
+edit_topsites_unpin_button=Unpin this site
 edit_topsites_edit_button=Edit this site
 edit_topsites_dismiss_button=Dismiss this site
+
+# LOCALIZATION NOTE (pocket): This is shown in the PocketStories component.
```

Comment:
This sounds more like a developer note. Can you explain what "PocketStories component" is?